Octokit/Helpers/AcceptHeaders.cs

```diff
@@ -21,5 +21,7 @@ public static class AcceptHeaders
         public const string SquashCommitPreview = "application/vnd.github.polaris-preview+json";
 
         public const string MigrationsApiPreview = "  application/vnd.github.wyandotte-preview+json";
+
+        public const string Reactions = "application/vnd.github.squirrel-girl-preview";
```

Comment:
Can you please rename this to `ReactionsPreview` or similar (to indicate it's a preview functionality)